src/test/java/bit/javaoop/FixedBudgetManagerTest.java

```diff
@@ -0,0 +1,19 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FixedBudgetManagerTest {
+
+    @Test
+    void canHire() {
```

Comment:
+ za zachowanie konwencji //given//when//then , ale jeszcze nazwy metod mógłbyś poprawić. Intellij takie sugeruje, ale lepiej trzymać się konwencji `shouldCosTamCosTam`, czyli np. w naszym przypadku `shouldHireEmployee`.

Ale tak ogólnie to test wg mnie zajebisty :) Możnaby jedynie pomyśleć nad wyciągnięciem czegoś do funkcji `beforeEach()`, ale w tak małym teście myślę, że nie jest to konieczne.